src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div class="ec-checkbox">
+    <input
+      :id="id"
+      v-model="inputModel"
+      :aria-describedby="errorId"
+      type="checkbox"
+      class="ec-checkbox__input"
+      :disabled="disabled"
+    >
+    <label
+      :for="id"
+      class="ec-checkbox__label"
+      :class="{
+        'ec-checkbox__label--has-error': isInvalid || $slots['error-message'],
+        'ec-checkbox__label--large': large
+      }"
+    >
+      <span v-if="labelMessage || $slots['label-message']">
+        {{ labelMessage }}
+        <slot name="label-message" />
+      </span>
+    </label>
+
+    <div
+      :id="errorId"
+      v-if="isInvalid || $slots['error-message']"
```

Comment:
you can just do this:
```
<slot name="error-message">{{errorMessage}}</slot>
```
^^ if slot is passed, it will be used, otherwise it till fallback to errorMessage prop.

Same trick can be use for label.

Reply:
I need the div